### PR TITLE
Peacekeepers and Security Pouches

### PR DESCRIPTION
**Pull request by AshTheDerg**
# About The Pull Request

Adds a new RP-Focused Security Role: Peacekeepers, alongside a new Security Pouch for small items such as Flashes, Cuffs, and Donuts

## Why It's Good For The Game

More RP-Focused roles are good for the game, especially ones focused on learning non-violent conflict resolution. Security Pouches will drastically reduce used space in officer bags.

## A Port?

No

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added Peacekeeper Role
add: Added Security Pouch
add: Added Peacekeeper Role Spawns
/:cl:

### CODE REVIEW

**Comment by radar651** [comment]
I do not mean to be that guy but this can and will cause problems with normal SecOffs as Peacekeeper is an alt tittle. This means icons will get overridden by the higher call. You also lack SecHud and ID icons so this may cause some issues with Officers trying to ID a person by a glance. It also looks bad having a generic ID and Grey out HUD.

File: modular_splurt/code/game/object/items/storage/bags.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/obj/item/storage/bag/security/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.max_combined_w_class = INFINITY
+	STR.max_items = 5
+	STR.display_numerical_stacking = TRUE
+	STR.can_hold = typecacheof(list(/obj/item/restraints/handcuffs, /obj/item/restraints/handcuffs/cable/zipties, /obj/item/assembly/flash,
+	/obj/item/reagent_containers/food/snacks/donut, /obj/item/melee/classic_baton/telescopic, /obj/item/grenade/flashbang, /obj/item/grenade/smokebomb, /obj/item/device/hailer, /obj/item/detective_scanner, /obj/item/reagent_containers/peacehypo, /obj/item/reagent_containers/spray/pepper))
```
Comment (radar651) [inline modular_splurt/code/game/object/items/storage/bags.dm:18]: This is EXTREMELY broken in the terms of balance as its a SecBelt+

Comment (Toxici11i) [reply modular_splurt/code/game/object/items/storage/bags.dm:18]: this literally holds more types of items than the original secbelt

lulw?

**Comment by radar651** [comment]
> > I do not mean to be that guy but this can and will cause problems with normal SecOffs as Peacekeeper is an alt tittle. This means icons, access, gear, ect will get overridden by the higher call. You also lack SecHud and ID icons so this may cause some issues with Officers trying to ID a person by a glance. It also looks bad having a generic ID and Grey out HUD.
> 
> Alt Titles won't affect gear, access, or icons. Alt Titles are just strings, they aren't the IDs. The IDs are what determines gear, access, and icons. As far as the ID and HUD issues, those can be changed.

Except they have in the past. The Engineer and Robotcist used to share Mechanic and sometimes Robotcist would spawn as engineers.

**Comment by AshTheDerg** [comment]
Nobody is happy with this. Closing the PR.

**Comment by AshTheDerg** [comment]
Main complaint was T-Baton. Brig Docs get one and no-one complains. Reopening.

**Pull request closed without merging**